setup.cfg

```diff
@@ -39,6 +39,7 @@ filterwarnings =
 # This is a temporary measure, all of these should be fixed:
     ignore::pytest.PytestUnraisableExceptionWarning
     ignore::numpy.VisibleDeprecationWarning
+    ignore::DeprecationWarning
```

Comment:
We shouldn't do this as we need to catch deprecations that we trigger with astroquery code.

Reply:
ach, yes, this was a mistake: for reasons I don't understand, I have to have this for tests to pass locally, but I didn't mean to commit it.